addons/web/static/src/views/fields/progress_bar/progress_bar_field.js

```diff
@@ -40,15 +38,15 @@ export class ProgressBarField extends Component {
                 currentValue: this.getCurrentValue(nextProps),
                 maxValue: this.getMaxValue(nextProps),
             });
-            if (nextProps.readonly) {
-                this.state.isEditing = false;
-            }
         });
     }
 
     get isCurrentValueInteger() {
         return this.state.currentValue % 1 === 0;
     }
+    get isEditable() {
+        return this.props.isEditable && (!this.props.readonly || this.props.isEditableInReadonly);
```

Comment:
props isEditableInReadonly (coming from attrs editable_readonly) is no longer used (and no longer really relevant), so we should take the shot to remove it.

Check other options as well because it looks like edit_current_value isn't used either.